asif/Data Mini Project.py

```diff
@@ -0,0 +1,234 @@
+import time
+import os
+from urllib.parse import parse_qsl
+#TODO Create a courier menu
+
+os.system("cls")
```

Comment:
```suggestion
CLEAR = "clear" if os.name == "posix" else "cls"
os.system(CLEAR)
```

Reply:
CLEAR constant to be able to detect OS type so system.os("cls") works on linux and mac as well as windows